### DISCUSSION TITLE
Manual sorting of collection applicable in Collection widget?

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#8647] [Troubleshooting] [answered]
Hi!

The docs for the Collection widget say:

"Sort and order by field name. E.g. 'title:desc'. Defaults to the collection's settings."

I have a collection which can be sorted manually, and I want to have the same sort order reflected in the widget. Is that part of "defaults to the settings"?

Or is there any other way to specify in cp.php that the widget should use that manually sort order?

Thanks!

PS: I also asked that via support email - I'll post here if I get an answer. Should have tried only here first, sorry!

**Comment by ryanmitchell** [comment, marked as answer]
If you want to sort by order it should be 'order:asc' or 'order:desc'. 
I would have thought it defaults to 'order:asc' when its a sortable collection.

**Reply by jensolafkoch** [reply]
Yep, thanks, that's what I was looking for! :-) I have to explicitly set `sort => 'order:asc'` in the widget's definition.
